content/docs/manage/backups.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Backups
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon does not yet provide support for configuring automated backups in the Neon Console or API. This feature is on our roadmap. You can expect it to be introduced in the coming months.  
+
+In the meantime, we support the following backup options:
+
+## Built-in backups with Neon's point-in-time restore feature
+
+By default, Neon keeps a 7-day history allowing you to restore your data to a particular date and time or Log Sequence Number (LSN). For Neon [Free Tier](/docs/introduction/free-tier) users, the 7-day history period is fixed. [Neon Pro plan](/docs/introduction/pro-plan) users can configure Neon to retain up to a 30 days of history. With this backup option, no action or automation is required. A data history is always available. You can quickly restore your data to a past state by creating a database branch. This feature is referred to point-in-time restore (PITR).
+
+For information about creating a point-in-time restore branch, see [Branching — Point-in-time restore (PITR)](/docs/guides/branching-pitr).
+
+## pg_dump
+
+You can backup a database using `pg_dump`, in the same way backups are created for a standalone Postgres instances.
```

Comment:
this should highlight that this approach backs the data from a single branch, so this needs to be repeated multiple times if the data from multiple branches needs to be downloaded

Reply:
and it will affect the traffic consumption